docs/USAGE-GUIDE.md

```diff
@@ -1361,7 +1361,7 @@ ssize_t s2n_client_hello_get_extension_by_id(struct s2n_client_hello *ch, s2n_tl
 - **max_length** Max number of bytes to copy into the **out** buffer.
 
 **s2n_client_hello_get_extension_length** returns the number of bytes the given extension type takes on the ClientHello message received by the server; it can be used to allocate the **out** buffer.
-**s2n_client_hello_get_extension_by_id** copies into the **out** buffer **max_length** bytes of a given extension type on the ClientHello and returns the number of bytes that were copied.
```

Comment:
Why are we updating this phrasing everywhere as part of this PR?

Reply:
Well, Laura suggested to update to this wording and I wanted to keep it even across the doc,  and it wasn't a huge effort to have consistency - so updated it.